### PR TITLE
Shard Hosting Presence.

### PR DESCRIPTION
**Pull request by Rhbarber**
Website: https://shardhosting.co.uk/

Activity Preview: Activity
Website Preview: Website

### CODE REVIEW

File: websites/S/Shard Hosting/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      "id": "732299294098587709"
+    }
+  ],
+  "url": "shardhosting.co.uk",
+  "description": {
+    "en": "Bringing your ideas to life in the most brilliant way!",
+    "es": "¡Hacemos realidad tus ídeas de la manera más brillante!"
+  },
+  "service": "Shard Hosting",
+  "version": "1.0",
```
Comment (EncryptedDev) [inline websites/S/Shard Hosting/dist/metadata.json:19]: Semver please
```suggestion
  "version": "1.0.0",

```

**Review by Bas950 — DISMISSED** [review]
Logo must be 512x512 (square).
Thumbnail must be a banner not the logo

File: websites/S/Shard Hosting/presence.ts [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    } else if (document.location.hostname === "web-ore-01.shardhosting.co.uk") {
+        presenceData.smallImageKey = "cpanel";
+        presenceData.smallImageText = "cPanel";
+        presenceData.details = "Web Hosting";
+        presenceData.state = "Managing cPanel";
+    }
+
+    if (!presenceData.details) {
+        presenceData.details = "Server Hosting";
+        presenceData.state = "Starting at $0.75";
```
Comment (Slowlife01) [inline websites/S/Shard Hosting/presence.ts:123]: Remove these

Comment (Rhbarber) [reply websites/S/Shard Hosting/presence.ts:123]: May I ask why as this presence is based upon [Sparked Host's one](https://github.com/PreMiD/Presences/blob/main/websites/S/Sparked%20Host/presence.ts) which has these lines as well?

Comment (Slowlife01) [reply websites/S/Shard Hosting/presence.ts:123]: Will be removed

**Review by Slowlife01 — APPROVED** [review]
lgtm

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**